client/signup/steps/about/index.jsx

```diff
@@ -633,12 +643,17 @@ class AboutStep extends Component {
 }
 
 export default connect(
-	state => ( {
+	( state, ownProps ) => ( {
 		siteTitle: getSiteTitle( state ),
 		siteGoals: getSiteGoals( state ),
 		siteTopic: getSurveyVertical( state ),
 		userExperience: getUserExperience( state ),
+		siteType: getSiteType( state ),
 		isLoggedIn: isUserLoggedIn( state ),
+		shouldHideSiteGoals: 'onboarding' === ownProps.flowName,
```

Comment:
@niranjan-uma-shankar Just a note here on why I moved this down to props from `this.state`. It's safe to set and leave here. The Component only needs to know the value, and we don't update the value within the Component.

Reply:
@ramonjd That makes sense 👍